eclipse/src/saros/editor/internal/EditorAPI.java

```diff
@@ -268,27 +272,275 @@ public static IResource getEditorResource(IEditorPart editorPart) {
   }
 
   /**
-   * Returns the current text selection for given editor.
+   * Returns the line base selection values for the given editor part.
    *
-   * @param editorPart the editorPart for which to get the text selection.
-   * @return the current text selection. Returns {@link TextSelection#emptySelection()} if no text
-   *     selection exists.
+   * <p>The given editor part must not be <code>null</code>.
```

Comment:
There was a hidden null check that returned an empty selection instead of throwing an exception.

Reply:
Yeah, it is still there l.290. Will remove the `Objects.requireNonNull(...)` and update the javadoc.